build/Targets/Settings.targets

```diff
@@ -150,7 +150,8 @@
 
         <Otherwise>
           <PropertyGroup>
-            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25604-Preview4</VisualStudioBuildToolsNuGetPackagePath>
+            <!-- TODO: Detect Preview 4/5 versus RC -->
+            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25825-RC</VisualStudioBuildToolsNuGetPackagePath>
```

Comment:
Without a version specific check, preview 5 gives:

> C:\Users\KevinPi\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25825-RC\tools\VSSDK\Microsoft.VsSDK.targets(606,5): e
rror VSSDK1048: Error trying to read the VSIX manifest file "C:\Code\Roslyn\Open\Binaries\Obj\VisualStudioDiagnosticsWi
ndow\Debug\extension.vsixmanifest". Could not load file or assembly 'file:///C:\Program Files (x86)\Microsoft Visual St
udio\VS15Preview\Common7\IDE\Microsoft.VisualStudio.ExtensionEngine.dll' or one of its dependencies. The system cannot
find the file specified. [C:\Code\Roslyn\Open\src\VisualStudio\VisualStudioDiagnosticsToolWindow\VisualStudioDiagnostic
sWindow.csproj]

I wouldn't feel *too* bad about just breaking people using Preview 5 and making them get an RC build though...